client/web/src/components/series/Info/SeriesInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { Paper, Box, IconButton, Divider, makeStyles } from '@material-ui/core'
+import BackIcon from '@material-ui/icons/ArrowBack'
+import { useTranslation } from 'react-i18next'
+import { actionTypes, selectors } from '../../../features/series'
+import { ISeries } from '../../../models/series'
+import { Toolbar } from '../../common'
+import { ILabel } from '../../../models/ILabel'
+
+import style from './SeriesInfo.style'
+
+type SeriesInfoProps = {
+  id: string
+}
+
+const useStyle = makeStyles(style)
+
+export const SeriesInfo: React.FC<SeriesInfoProps> = (
+  props: SeriesInfoProps
+) => {
+  const classes = useStyle()
+  const { id } = props
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const series: ISeries = useSelector(selectors.getCurrent)
+  const { t } = useTranslation()
+
+  useEffect(() => {
+    dispatch({ type: actionTypes.FETCH_ONE_SERIES, id })
```

Comment:
I don't see the need of fetch the series ? all information are in const series: ISeries = useSelector(selectors.getCurrent) no ?

Reply:
useSelector(selectors.getCurrent) returns the last result of fetch the series. It is better to call a fetch for having the last fresh data.